codesyncapp/codesync-vs
# Using await glob instead of globSync, Using lock update time as 1000ms

**Testing Instructions**
- For large repo, if populateBuffer takes longer than expected, bufferHandler should keep sending diffs since we are now using async/await version of glob
- Locks should work fine across multiple instances. If 1 instance with acquired locks is closed, other open instance should acquire those locks to make populateBuffer and bufferHandlers work properly
